x-pack/plugins/secops/public/components/timeline/body/sort/sort_indicator.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import * as React from 'react';
+import { pure } from 'recompose';
+import { SortDirection } from './sort';
+
+export const SORT_DIRECTION_ASCENDING = '^';
+export const SORT_DIRECTION_DESCENDING = 'v';
+export const SORT_DIRECTION_NONE = '';
+
+/** Returns the symbol that corresponds to the specified `SortDirection` */
+export const getDirection = (sortDirection: SortDirection): string => {
+  switch (sortDirection) {
+    case 'ascending':
+      return SORT_DIRECTION_ASCENDING;
+    case 'descending':
+      return SORT_DIRECTION_DESCENDING;
+    default:
```

Comment:
I'd add an explicit case for `''` to be more exact branch choice. Otherwise seems like I can malfunction the code and pass in whatever I want and it just does this little ole' default here. Seems so ... unsafe ;-) this JavaScript type coding.

That `default` should be a programmer error if there is only 1 of 3 possible states.